bin/cli.js

```diff
@@ -38,14 +40,16 @@ if (argv.h) {
   console.log('')
   console.log('Options:')
   console.log('')
-  console.log('  -d, --diff             output diff against original file')
-  console.log('  -l, --list             format list of space seperated files in place')
-  console.log('  -R, --recursive        format files recursively')
-  console.log('  -c, --config           path to a specific configuration file (JSON, YAML, or CommonJS)')
-  console.log('  -b, --config-basedir   path to the directory that relative paths defining "extends"')
-  console.log('  -v, --version          output the version number')
-  console.log('  -h, --help             output usage information')
-  process.exit()
+  console.log('  -d, --diff             Output diff against original file')
+  console.log('  -l, --list             Format list of space seperated files in place')
+  console.log('  -R, --recursive        Format files recursively')
+  console.log('  -c, --config           Path to a specific configuration file (JSON, YAML, or CommonJS)')
+  console.log('  -b, --config-basedir   Path to the directory that relative paths defining "extends"')
+  console.log('  -v, --version          Output the version number')
+  console.log('  -h, --help             Output usage information')
+  console.log('  -i, --ignore-path      Path to a file containing patterns that describe files to ignore.')
+  console.log('  -id --ignore-disables  Ignore disables')
+  console.log('  --stdin-filename       A filename to assign stdin input.')
```

Comment:
This can be useful, for example, when making a text editor plugin that passes in code directly but needs to still use the configuration's ignoreFiles functionality to possibly ignore that code.

See:
https://github.com/stylelint/stylelint/blob/master/src/cli.js#L155
http://stylelint.io/user-guide/node-api/#codefilename